docs/acl.rst

```diff
@@ -8,4 +15,74 @@ Control features within Fledge have the ability to add access control to every s
 Graphical Interface
 ~~~~~~~~~~~~~~~~~~~
 
-A graphical interfaces is available to allow the creation and management of the access control lists used within the Fledge system. This is available within the *Control* menu item of the Fledge graphical interface.
+A graphical interfaces is available to allow the creation and management of the access control lists used within the Fledge system. This is available within the *Control Dispatcher* menu item of the Fledge graphical interface.
+
++---------+
+| |ACL_1| |
++---------+
+
+Adding An ACL
+#############
+
+Click on the *Add* button in the top right corner of the ACL screen, the following screen will then be displayed.
+
++---------+
+| |ACL_2| |
++---------+
+
+You can enter a name for your access control list in the *Name* item on the screen. You should give each of your ACLs a unique name, this may be anything you like, but should ideally be descriptive or memorable as this is the name you will use when associating the ACL with services and scripts.
+
+The *Services* section is use to define a set of services that this ACL is allowing access for. You may select services either by name or by service type. Multiple services may be granted access by a single ACL.
+
++---------+
+| |ACL_3| |
++---------+
+
+To add a named service to the ACL select the names drop down list and select the service name from those displayed. The display will change to show the service that you added to the ACL.
+
++---------+
+| |ACL_4| |
++---------+
+
+More names may be added to the ACL by selecting the drop down again.
+
++---------+
+| |ACL_5| |
++---------+
+
+If you wish to remove a named service from the list of services simply click on the small *x* to the left of the service name you wish to remove.
+
+It is also possible to add a service type to an ACL. In this case all services of this type in the local Fledge instance will be given access via this ACL.
+
++---------+
+| |ACL_6| |
++---------+
+
+For example to create an ACL that allows all north services to have be granted access you would select *Northbound* in the *Services Types* drop down list.
+
++---------+
+| |ACL_7| |
++---------+
+
+The *URLs* section of the ACL is used to grant access to specific URLs accessing the system. This is intended to allow control access via the REST API of the Fledge instance and is currently not implemented in Fledge. 
```

Comment:
We may need to make highlight "This is intended to allow control access via the REST API of the Fledge instance and is currently not implemented in Fledge". Probably with note directive. `..note::`